dbt-greenery/models/marts/product/fct_product_session_v2.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(materialized='table') }}
+
+{% set event_types = [
+    'page_view'
+    , 'add_to_cart'
+    , 'delete_from_cart'
+]
+ %}
+
+
+WITH 
+    
+
+    {% for event_type in event_types %}
+      {{event_type}}_table AS (
+      {{funnel_step(event_type,int_event_log)}})
```

Comment:
```suggestion
      {{funnel_step(event_type, ref('int_event_log')  )}})
```

Reply:
You need to use ref here so that dbt replaces it with the tablename that int_event_log refers to ... right now it is looking for a variable called `int_event_log` which doesn't exist.